README.md

```diff
@@ -1,3 +1,21 @@
 [![CircleCI](https://circleci.com/gh/adessoAG/adessoKicker/tree/master.svg?style=svg)](https://circleci.com/gh/adessoAG/adessoKicker/tree/master)
-# adessoKicker
-Program to organize foosball tournaments and teams.
+
+# What is adesso kicker?
+Adesso kicker is a website for adesso coworkers to make playing foosball even more fun and to find out who is the best kicker player.
+Each person can login with Keycloak and record the result of matches they have played.
+Our website is sending notifications and email to the other participants, so they can
+confirm valid games.
+Our internal ranking system based on elo will determine the position and skill of each person.
```

Comment:
Our internal ranking system based on elo will determine the skill of each person and position them into and ranking list.